src/pages/History/History.jsx

```diff
@@ -0,0 +1,29 @@
+import "./history.css";
+
+import { useEffect } from "react";
+
+import useVideoLib from "../../context/videos-context";
+import VideoCard from "../Home/VideoCard";
+
+export default function History() {
+  const { state } = useVideoLib();
+
+  const watchHistory = state.history;
```

Comment:
instead of making new variable as watchHistory you can directly destructure history from state as 
const { state : { history} } = useVideoLib()

Reply:
was trying to reverse the history array, but then achieved it by css and this has leftover

Reply:
yes